Content.Server/Medical/CryoPodSystem.cs

```diff
@@ -55,6 +55,9 @@ public sealed partial class CryoPodSystem : SharedCryoPodSystem
     [Dependency] private readonly IAdminLogManager _adminLogger = default!;
     [Dependency] private readonly NodeContainerSystem _nodeContainer = default!;
 
+    // Frontier: keep a list of cryogenics reagents. The pod will only filter these out from the provided solution.
+    private static readonly string[] CryogenicsReagents = ["Cryoxadone", "Aloxadone", "Doxarubixadone", "Opporozidone", "Necrosol", "Traumoxadone", "Stelloxadone"];
```

Comment:
Don't like this: frankly, it should be searchable by MetabolismGroup without LINQ.

Whenever the reagents get loaded from YAML, you could look through the effects, associate them with the metabolism groups they have effects for, and then access that set in some lifecycle call here to build this array.

Reply:
i would recommend not using a hard coded list.  it will make it makes it harder to maintain given somebody might add, re-name, remove, update, something in the future.